arwen/contexts/runtime.go

```diff
@@ -257,7 +257,7 @@ func (context *runtimeContext) ResetWarmInstance() {
 		return
 	}
 
-	arwen.RemoveHostContext(*context.instance.Data)
+	arwen.RemoveHostContext(*context.instance.GetData())
```

Comment:
is it safe to get the pointer and remove this like this ? make sure GetData returns what it needs.

Reply:
Tested, and it seems all right.